torch/_ops.py

```diff
@@ -971,6 +1003,19 @@ def __call__(self_, *args, **kwargs):  # noqa: B902
         # is still callable from JIT
         # We save the function ptr as the `op` attribute on
         # OpOverloadPacket to access it here.
+
+        # Directly calling OverloadPacket goes into C++, which will check
+        # the schema and cause an error for torchbind op when inputs FakeScriptObject so we
+        # intercept it here and call TorchBindOpverload instead.
+        if self_._has_torchbind_op_overload:
+            # Try the overloads one by one to match an op.
+            for overload_name in self_._schemas.keys():
+                try:
+                    ret = getattr(self_, overload_name)(*args, **kwargs)
+                except RuntimeError as e:
+                    continue
+                return ret
```

Comment:
Try to reuse the existing C++ logic as much as possible

Reply:
Just checked that the overload_names are sorted already. _jit_get_operation returns a cpp func and all its overload_names in sorted order see [here](https://github.com/pytorch/pytorch/blob/main/torch/csrc/jit/python/init.cpp#L1697-L1714). So we're OK to try in this order?